app/controllers/api/v0/reactions_controller.rb

```diff
@@ -25,14 +25,6 @@ def create
         render json: { reaction: @reaction.to_json }
       end
 
-      def onboarding
-        verify_authenticity_token
-        reactable_ids = JSON.parse(params[:articles]).map { |article| article["id"] }
-        reactable_ids.each do |article_id|
-          Reactions::CreateWorker.perform_async(current_user.id, article_id, "Article", "readinglist")
```

Comment:
Can we also remove this worker then?

Reply:
Yes, we can! Good catch, I totally forgot about that, since originally this PR just moved code. Thanks 🙏